packages/block-library/src/site-description/edit.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import {
+	AlignmentToolbar,
+	__experimentalBlock as Block,
+	BlockControls,
+	RichText,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+export default function SiteDescriptionEdit( { attributes, setAttributes } ) {
+	const { align } = attributes;
+	const [ description, setDescription ] = useEntityProp(
+		'root',
+		'site',
+		'description'
+	);
+
+	return (
+		<>
+			<BlockControls>
+				<AlignmentToolbar
+					onChange={ ( newAlign ) =>
+						setAttributes( { align: newAlign } )
+					}
+					value={ align }
+				/>
```

Comment:
I think [adding `align` to `suppports`](https://github.com/WordPress/gutenberg/pull/23788/files#r452192964) will give you the `AlignmentToolbar` out f the box :crossed_fingers:

Reply:
@ockham The difference is that `align` in `supports` is the _block_ alignment, while in this case it would be the _text_ alignment.
Two major differences, off the top of my head:
- Block alignment also (optionally) supports `wide` and `full`.
- Block alignment `left` and `right` makes the block floating.

This said, your point makes sense in that we might want this block to support both block (as `align` `supports`) and text alignment (as `textAlign` attribute).
I don't see any other Core block doing the same thing, and I wonder if it's an intentional UX choice (it might be confusing for the user).
A middle ground could be to have `align` only support `wide, full`. 🤔

Reply:
Right, thanks for clarifying.

I do think that this has the potential of being quite confusing to the user, so I wouldn't be surprised if it was a deliberate UX choice.

Do blocks that are somewhat comparable to Site Description tend to have either align or text align?

If we do go with text align, let's rename the attribute to `textAlign`, to make it easier to distinguish from block align.

Reply:
The most comparable is arguably [Site Title](https://github.com/WordPress/gutenberg/blob/5dbff73802fe5c50bf0489dfee7d9bb7f44589a7/packages/block-library/src/site-title/edit.js), which indeed only has the text alignment with the `align` attribute name.

(It's not a coincidence, I've mostly copied it to build the Site Description block... 😄)

Reply:
> If we do go with text align, let's rename the attribute to textAlign, to make it easier to distinguish from block align.

I agree, but I'd love more Gutenberg folks chime in on this.
As far as I can see, only Verse uses `textAlign` for text alignment, while all other blocks simply use `align`.